.github/CONTRIBUTING.md

```diff
@@ -16,27 +16,29 @@ We are always looking for help implementing new features or fixing bugs.
    - Add details on how to reproduce the issue - a minimal test case is always best, colab is also great.
      Note, that the sample code shall be minimal and if needed with publicly available data.
 
-2. Try to fix it or recommend a solution. We highly recommend to use test-driven approach:
+1. Try to fix it or recommend a solution. We highly recommend to use test-driven approach:
 
    - Convert your minimal code example to a unit/integration test with assert on expected results.
    - Start by debugging the issue... You can run just this particular test in your IDE and draft a fix.
    - Verify that your test case fails on the master branch and only passes with the fix applied.
 
-3. Submit a PR!
+1. Submit a PR!
 
 _**Note**, even if you do not find the solution, sending a PR with a test covering the issue is a valid contribution and we can
-help you or finish it with you :]_
+help you or finish it with you :\]_
 
 ### New Features:
 
 1. Submit a github issue - describe what is the motivation of such feature (adding the use case or an example is helpful).
-2. Let's discuss to determine the feature scope.
-3. Submit a PR! We recommend test driven approach to adding new features as well:
+
+1. Let's discuss to determine the feature scope.
+
+1. Submit a PR! We recommend test driven approach to adding new features as well:
```

Comment:
Lot's of erros with numberred lists haha

Reply:
there are no errors :D